apps/condo/domains/common/utils/serverSideBankAccountApi.js

```diff
@@ -0,0 +1,32 @@
+const get = require('lodash/get')
+const fetch = require('node-fetch')
+const conf = require('@condo/config')
+
+const bankAccountSuggestionsConfig = conf['BANK_ACCOUNT_SUGGESTIONS_CONFIG'] && JSON.parse(conf['BANK_ACCOUNT_SUGGESTIONS_CONFIG'])
+const API_URL_BANK = get(bankAccountSuggestionsConfig, 'apiUrlBank', null)
+const API_TOKEN = get(bankAccountSuggestionsConfig, 'apiToken', null)
+
+async function getBankByRoutingNumber (query) {
```

Comment:
Where this function is used?
It look so abstract…
Are you sure, that every API will receive `query` parameter?